editor/plugins/animation_state_machine_editor.cpp

```diff
@@ -1601,6 +1623,205 @@ void AnimationNodeStateMachineEditor::_update_mode() {
 	}
 }
 
+void AnimationNodeStateMachineEditor::_node_group_edit_redraw(StringName node_name) {
+	if (side_panel_mode == NODE_GROUP_EDIT) {
+		if (bool(node_name)) {
+			sidepanel_title_2->set_text(String(node_name));
+			sidepanel_title_2->add_theme_color_override("font_color", Color(.55, .93, .59));
```

Comment:
Did you mean StringName for font color type like; StringName("font_color") ?

Reply:
No I meant exactly what I wrote, you need to update your outdated branch and apply this one correctly

Reply:
When I try to compile this it gives, `error: 'class SceneStringNames' has no member named 'font_color', which is also the case when I check SceneStringNames nothing about font in there. Also there is no example of `SceneStringName(font_color)` in whole code base, as far as I can tell.

Reply:
Yes, like I said, you need to update your branch, you're three weeks behind and you don't have the latest updates, so please update your branch and then try, use git rebase -i upstream master

Reply:
Ah, sorry, I get it now, didn't expect things to change so fast..

Reply:
We have dozens of PRs merged every week! On a slow week! Always make sure you at least start a PR on an up to date branch

But most importantly, if you're confused or unsure about something, wait for a response (I almost missed this one because you marked it as resolved when it wasn't) :)

Reply:
I was so sure xD